spec/requests/form_controller_spec.rb

```diff
@@ -314,6 +314,12 @@
           get "/lettings-logs/#{setup_complete_lettings_log.id}/review", headers: headers, params: {}
           expect(response.body).to match("Review lettings log")
         end
+
+        it "renders the review page for the sales log" do
+          log = create(:sales_log, :completed, created_by: user)
+          get "/sales-logs/#{log.id}/review", headers: headers, params: { sales_log: true }
```

Comment:
are we passing query params to determine the log type? is the log type not already given earlier in the path eg `/sales-log` or `/lettings-log`

Reply:
sure, i feel like this will stick around and fester until it's addressed though